pkg/controllers/routing/network_routes_controller.go

```diff
@@ -1098,107 +1135,30 @@ func (nrc *NetworkRoutingController) startBgpServer(grpcServer bool) error {
 	// If the global routing peer is configured then peer with it
 	// else attempt to get peers from node specific BGP annotations.
 	if len(nrc.globalPeerRouters) == 0 {
-		// Get Global Peer Router ASN configs
-		nodeBgpPeerAsnsAnnotation, ok := node.Annotations[peerASNAnnotation]
-		if !ok {
-			klog.Infof("Could not find BGP peer info for the node in the node annotations so " +
-				"skipping configuring peer.")
-			return nil
-		}
-
-		asnStrings := stringToSlice(nodeBgpPeerAsnsAnnotation, ",")
-		peerASNs, err := stringSliceToUInt32(asnStrings)
+		klog.V(2).
+			Infof("Attempting to construct peer configs from annotation: %+v", node.Annotations)
```

Comment:
2 things here... In general, I try never to split lines on chained functions in the kube-router code base unless there is no other option. In this case, I think it would be better to either wrap the parameter `node.Annotations` or if that doesn't get it below the char limit, then I would split the string across multiple lines.

I'm a bit iffy on this. Technically tier 2 logging should probably only be used for debugging, but technically this can also leak BGP passwords in logs. I could see going both ways on this, but usually its better to lean more secure even if that gives people less debug info.

Reply:
Same here for breaking the line during function chaining.

The more that I think about it, I don't think that logging the full annotation when debugging is increased to level 2 is a big deal. People should expect some more "secret" type information to get displayed when they get into deep debugging levels.